rct229/rulesets/ashrae9012019/section16/section16rule2.py

```diff
@@ -0,0 +1,85 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
+from rct229.rulesets.ashrae9012019 import BASELINE_0, PROPOSED, USER
+from rct229.rulesets.ashrae9012019.ruleset_functions.compare_schedules import (
+    compare_schedules,
+)
+from rct229.utils.assertions import getattr_
+
+
+class Section16Rule2(RuleDefinitionListIndexedBase):
+    """Rule 2 of ASHRAE 90.1-2019 Appendix G Section 16 (Elevators)"""
+
+    def __init__(self):
+        super(Section16Rule2, self).__init__(
+            rmds_used=produce_ruleset_model_instance(
+                USER=False,
+                BASELINE_0=True,
+                PROPOSED=True,
+            ),
+            each_rule=Section16Rule2.BuildingRule(),
+            index_rmd="baseline",
+            id="16-2",
+            description="The baseline elevator motor use shall be modeled with the same schedule as the proposed design. Rule Assertion: B-RMD = P-RMD",
+            ruleset_section_title="Elevators",
+            standard_section="Section G3.1",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].buildings[*]",
+            required_fields={"$": ["calendar"], "$.calendar": ["is_leap_year"]},
+            data_items={"is_leap_year_b": (BASELINE_0, "calendar/is_leap_year")},
+        )
+
+    class BuildingRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section16Rule2.BuildingRule, self).__init__(
+                rmds_used=produce_ruleset_model_instance(
+                    USER=False,
+                    BASELINE_0=True,
+                    PROPOSED=True,
+                ),
+            )
+
+        def is_applicable(self, context, data=None):
+            building_p = context.PROPOSED
+            elevators_p = building_p.get("elevators", [])
+
+            return len(elevators_p) > 0
```

Comment:
is_applicable looks to be checking if the rule is applicable for each building in the RMD. The RDS was written so that the rule is only not applicable if none of the buildings in the RMD have an elevator (If there are 3 buildings in the RMD and only 1 building has an elevator then the rule is still applicable)

I don't think there is a strong preference on this so if there is a good reason to do it differently please let me know.

Reply:
Good point. Addressed.